src/pages/api/auth/[...nextauth].ts

```
@@ -30,6 +31,31 @@ export const authOptions: NextAuthOptions = {
+            "https://people.googleapis.com/v1/people/me?personFields=photos",
+            {
+              headers: {
+                Authorization: `Bearer ${account.access_token}`,
+                Accept: "application/json",
+              },
+            },
+          );
+
+          if (response.ok) {
+            const data = await response.json();
+            if (data.photos && data.photos[0]?.url) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)